samples/starterKit/skill/MainStateMachine.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+// Include the state machine module and the replyWith function
+const Voxa = require('voxa');
+const views = require('./views');
+const variables = require('./variables');
+const states = require('./states');
+const voxaOpearlo = require('voxa-opearlo');
+
+const opearloConfig = {
+  userId: 'tkT8PClpPxMOvPbadGyzOoJKwoX2',
```

Comment:
this shouldn't be here! let's add this to a configuration key and leave it empty with instructions in the readme about it